roles/remove-node/post-remove/tasks/main.yml

```diff
@@ -1,9 +1,9 @@
 ---
 - name: Remove-node | Delete node
   command: "{{ kubectl }} delete node {{ kube_override_hostname | default(inventory_hostname) }}"
-  delegate_to: "{{ groups['kube_control_plane'] | first }}"
+  delegate_to: "{{ groups['kube_control_plane'] | community.general.lists_difference(node) | first }}"
   when:
-    - groups['kube_control_plane'] | length > 0
+    - groups['kube_control_plane'] | community.general.lists_difference(node) | length > 0
```

Comment:
If this means to avoid removing the last control plane ? If so that probably deserve a comment